package.json

```diff
@@ -17,38 +17,41 @@
     "fix-package-lock": "rm -f package-lock.json && rm -rf node_modules && npm i"
   },
   "devDependencies": {
+    "@eslint/js": "^9.9.0",
     "@playwright/test": "^1.51.1",
     "@sveltejs/adapter-static": "^3.0.8",
-    "@sveltejs/kit": "^2.20.2",
     "@sveltejs/vite-plugin-svelte": "^5.0.3",
     "@types/eslint": "9.6.1",
     "@types/node": "^22.13.14",
     "@typescript-eslint/eslint-plugin": "^8.28.0",
-    "@typescript-eslint/parser": "^8.28.0",
+    "@typescript-eslint/parser": "^8.31.1",
     "autoprefixer": "^10.4.21",
     "eslint": "9.23.0",
-    "eslint-config-prettier": "^10.1.1",
     "eslint-plugin-svelte": "^3.3.3",
     "jsdom": "^26.0.0",
     "postcss": "^8.5.3",
     "prettier": "^3.5.3",
+    "prettier-plugin-organize-imports": "^4.1.0",
     "prettier-plugin-svelte": "^3.3.3",
     "prettier-plugin-tailwindcss": "^0.6.11",
-    "svelte": "^5.25.3",
-    "svelte-check": "^4.1.5",
-    "tailwindcss": "^3.4.17",
-    "tailwindcss-debug-screens": "^2.1.0",
-    "tslib": "^2.8.1",
-    "typescript": "^5.8.2",
+    "svelte": "^5.0.0",
```

Comment:
This is a version downgrade as are the vite and typscript versions, but the PR description says upgrade . . ?